tests/e2e/tests/build/scripts-array.ts

```diff
@@ -48,5 +50,7 @@ export default function () {
       <script type="text/javascript" src="scripts.bundle.js"></script>
       <script type="text/javascript" src="vendor.bundle.js"></script>
       <script type="text/javascript" src="main.bundle.js"></script>
-    `));
+    `))
+     // ensure scripts aren't using script-loader when imported from the app
+    .then(() => expectFileToMatch('dist/main.bundle.js', 'console.log(\'string-script\');'));
```

Comment:
Looks to me that the real fix isn't this PR then. Can we get rid of script-loader?

I don't want people having 2 versions of, say, jQuery in their apps... we'll have tons of issues filed with that.